controllers/pipelinerun_controller.go

```diff
@@ -0,0 +1,203 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/shipwright-io/build/pkg/apis/build/v1alpha1"
+	"github.com/shipwright-io/triggers/pkg/filter"
+	"github.com/shipwright-io/triggers/pkg/inventory"
+	"github.com/shipwright-io/triggers/test/stubs"
+
+	"github.com/go-logr/logr"
+	tknv1beta1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
```

Comment:
Similar as above (and I know we have never discussed this), but given we know that Tekton moves to v1 soon, I suggest to use an alias like this:

```suggestion
	pipeline "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
```

Reply:
Let's tackle this on #9